src/style.css

```diff
@@ -0,0 +1,146 @@
+/* #region global */
```

Comment:
please, delete all comments (regions as well) before pushing changes
regions are used for your own comfort - only in the development stage